_search-plugins/hybrid-search.md

```diff
@@ -1212,4 +1212,4 @@ The response contains scoring information:
 
 Field | Description
 :--- | :---
-`explanation` | The `explanation` object has three properties: `value`, `description`, and `details`. The `value` property shows the result of the calculation, `description` explains what type of calculation was performed, and `details` shows any subcalculations performed. For score normalization, the information in the `description` property includes the technique used for normalization or combination and the corresponding score. 
+`explanation` | The `explanation` object has three properties: `value`, `description`, and `details`. The `value` property shows the result of the calculation, `description` explains what type of calculation was performed, and `details` shows any subcalculations performed. For score normalization, the information in the `description` property includes the technique used for normalization or combination and the corresponding score.
```

Comment:
How about create a new page of hybrid-search-rrf.md and add the relevant details there. Also show with examples how it is used . Also explain how search results are formed in terms ranking.

Reply:
I have the information about what it is and how to use it in the rrf processor page.  I don't think we need a third new page overall.  WDYT?